README.md

```diff
@@ -164,7 +164,7 @@ When you reach for this type of coercing you can expect a better behavior in the
 closedCoerceArray ∷ Effect Unit
 closedCoerceArray = do
   let
-    argument = { x: [] }
+    argument = { x: [] :: Array Int }
```

Comment:
Found a bug while running tests. CI is useful - who woulda thunk! :-)

Reply:
But this now contradicts the paragraph right above: it does not, in fact, work without a type annotation.

And to be clear, it _should_ logically work. It doesn't because of this issue: https://github.com/purescript/purescript/issues/4338

So I'm not sure how to fix this. Remove the whole "Pros" section? Or reference the issue maybe?

Reply:
I'm going to add this minor comment and tag a patch release today as well.